android/titanium/src/java/org/appcelerator/titanium/view/TiUIView.java

```diff
@@ -910,6 +896,17 @@ public void propertyChanged(String key, Object oldValue, Object newValue, KrollP
 			if (!nativeViewNull) {
 				nativeView.postInvalidate();
 			}
+		} else if (key.equals(TiC.PROPERTY_TOUCH_FEEDBACK_COLOR) || key.equals(TiC.PROPERTY_TOUCH_FEEDBACK)) {
+			KrollDict d = proxy.getProperties();
```

Comment:
before the logic called:
```java
 // Update first before querying.
proxy.setProperty(key, newValue);
```
before calling `proxy.getProperties();` Is it still necessary?

Reply:
From what I see the properties dictionary of the proxy is updated here:
https://github.com/appcelerator/titanium_mobile/blob/master/android/titanium/src/java/org/appcelerator/kroll/KrollProxy.java#L1125
This happens before calling the `propertyChanged`, so I think this line would not be necessary in any scenario. I see the line is from quite some time, so it may be a leftover from something not in use anymore?